src/libstore/derivations.cc

```diff
@@ -782,6 +787,8 @@ std::optional<BasicDerivation> Derivation::tryResolve(Store& store, const StoreP
     // This is quite dirty and leaky, but will disappear once #4340 is merged
     static Sync<std::map<StorePath, std::optional<Derivation>>> resolutionsCache;
 
+    debug("Trying to resolve %s", store.printStorePath(drvPath));
```

Comment:
```suggestion
    debug("trying to resolve %s", store.printStorePath(drvPath));
```

Messages are currently lower-case.